src/modules/dynamic-ui-wrapper/dynamic-ui-wrapper.ts

```diff
@@ -52,36 +51,20 @@ export class DynamicUiWrapper {
     this._activeSolutionEntry = solutionEntry;
   }
 
-  public async handleUserTaskButtonClick(action: 'cancel' | 'proceed' | 'decline'): Promise<void> {
-    const actionCanceled: boolean = action === 'cancel';
+  public async handleUserTaskButtonClick(action: ButtonClickActions, userTask: any): Promise<void> {
```

Comment:
Why is userTask any here?
I would expect that 
```suggestion
  public async handleUserTaskButtonClick(action: ButtonClickActions, userTask: DataModels.UserTasks.UserTask): Promise<void> {
```

Reply:
still so